youtube_dl/extractor/mtv.py

```diff
@@ -201,7 +201,7 @@ def _get_videos_info_from_url(self, url, video_id):
             [self._get_video_info(item) for item in idoc.findall('.//item')],
             playlist_title=title, playlist_description=description)
 
-    def _extract_mgid(self, webpage):
+    def _extract_mgid(self, webpage, fatal=True):
```

Comment:
Uses `default` as of https://github.com/rg3/youtube-dl/pull/11312/commits/9d60d1851a221f9369db40292f71cb56c12116d6.